common.mk

```diff
@@ -458,9 +458,25 @@ define launch-terminal
 		$(LAUNCH_TERMINAL) "$(BUILD_PATH)/soc_term.py $(1)" &
 endef
 else
+tmux := $(TMUX)
+tmux_window := $(shell echo OPTEE_TEST_$$RANDOM)
```

Comment:
I suggest `OPTEE_$$RANDOM` because it's shorter and more generic (`optee_test` is the name of the xtest project)